web/client/components/maps/MapCard.jsx

```diff
@@ -115,6 +119,17 @@ class MapCard extends React.Component {
                     this.onEdit(this.props.map, true);
                 }
             },
+            {
+                visible: this.props.shareToolEnabled === true,
+                glyph: 'share-alt',
+                disabled: this.props.map.updating,
```

Comment:
Hi, i would add this
```suggestion
                disabled: this.props.map && this.props.map.updating,
```